.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,101 @@
+name: PHP Coding Standards
+
+on:
+    push:
+        branches:
+            - trunk
+            - 'release/**'
+            - 'wp/**'
+    pull_request:
+        paths:
+            # Any change to a PHP file should run checks.
+            - '**.php'
```

Comment:
Is `.php` the only extension in use for PHP files ?

Reply:
That's a good question.
Some PHP code could be in stored .js files
But I removed the paths section in 
https://github.com/WordPress/gutenberg/commit/1d10c2c294cad20dcea822e785bc1d88c357420c